src/containers/Storage/Vdisk/Vdisk.js

```diff
@@ -225,33 +230,41 @@ function Vdisk(props) {
         const available = AvailableSize ? AvailableSize : PDisk?.AvailableSize;
 
         if (!available) {
-            return;
+            return undefined;
         }
-        return !isNaN(Number(AllocatedSize))
-            ? (Number(AllocatedSize) * 100) / (Number(available) + Number(AllocatedSize))
-            : undefined;
+
+        return isNaN(Number(AllocatedSize))
+            ? undefined
+            : (Number(AllocatedSize) * 100) / (Number(available) + Number(AllocatedSize));
```

Comment:
changes to satisfy the linter: fix inconsistent return type and a negated condition